src/librustc_errors/lib.rs

```diff
@@ -845,3 +845,11 @@ impl Level {
         }
     }
 }
+
+#[macro_export]
+macro_rules! pluralise {
+            ($x:expr) => {
```

Comment:
Dedent this. =P

Reply:
Ahh, missed that. Thanks @Centril!
Fixed in commit 7457ef8. :)